test/gtest/ucs/test_hash_perf.cc

```diff
@@ -0,0 +1,281 @@
+/**
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2016.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+#pragma GCC optimize("O3")
```

Comment:
We need high level optimization at all time to run this test because it has visible influence to performance results.